Test/Models/Extension/ViewExtension.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+
+namespace Test.Models.Extension
+{
+    public static class ViewExtension
+    {
+        public static void StartActivityVideoOrPdf(this View view, string path)
+        {

```

Comment:
Looks similar to a method from class `ActivityExtension `
You can do `view.Context. StartActivityVideoOrPdf(....)`

Reply:
Done